lib/diagnostic.js

```diff
@@ -32,6 +45,33 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.houses.update(
```

Comment:
This will not work because of the invalid syntax and the duplicate `$set` keys.  Here is a solution using a combination of `$set`, `$push` with a single argument, and `$push` an object argument with an `$each` key.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```